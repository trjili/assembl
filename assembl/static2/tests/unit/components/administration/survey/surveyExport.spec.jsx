import React from 'react';
import renderer from 'react-test-renderer';

import SurveyExport from '../../../../../js/app/components/administration/survey/surveyExport';

describe('SurveyExport component', () => {
  it('should render an admin section title', () => {
    const i18n = {
      locale: 'fr',
      translations: {
        en: {
          administration: {
            survey: {
              0: 'My section title'
            }
          }
        },
        fr: {
          administration: {
            survey: {
              0: 'Mon titre de section'
            }
          }
        }
      }
    };
    const component = renderer.create(<SurveyExport i18n={i18n} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});