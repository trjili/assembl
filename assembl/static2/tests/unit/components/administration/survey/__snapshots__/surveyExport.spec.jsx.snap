exports[`SurveyExport component should render an admin section title 1`] = `
<div
  className="hidden">
  <div>
    <h3
      className="dark-title-3" />
    <div
      className="box-hyphen" />
    <div
      className="annotation">
      Annotation
    </div>
  </div>
  <div
    className="admin-content">
    <a
      className="button-link button-dark margin-l"
      href="/data/Discussion/null/phase1_csv_export"
      onClick={[Function]}
      style={Object {}}>
      <span>
        Link
      </span>
    </a>
  </div>
</div>
`;
